CRM/Utils/Check/Component/Env.php

```diff
@@ -835,6 +835,25 @@ public function checkDbEngine() {
     return $messages;
   }
 
+  /**
+   * ensure reply id is set to any default value
+   * @return array
+   */
+  public function checkReplyIdForMailing() {
+    $messages = array();
+
+    if (!CRM_Mailing_PseudoConstant::defaultComponent('Reply', '')) {
+      $messages[] = new CRM_Utils_Check_Message(
+        __FUNCTION__,
+        ts('Reply Auto Responder is not set to any default value in <a href=%1>Headers, Footers, and Automated Messages</a>. This will disable the submit operation on any mailing created from CiviMail.', array(1 => CRM_Utils_System::url('civicrm/admin/component', 'reset=1'))),
+        ts('No Default value for Auto Responder.'),
+        \Psr\Log\LogLevel::WARNING,
+        'fa-server'
```

Comment:
@agh1 thinks that the reply icon "fa-reply" would be more fitting for this functionality

Reply:
Updated the PR to use `fa-reply` icon in the display. Thanks 👍